README.md

```diff
@@ -1,108 +1,86 @@
-# Application to demonstrate Mbed-OS crash reporting feature
+![](./resources/official_armmbed_example_badge.png)
+# Crash reporting Mbed OS example
 
-This example demonstrates how Mbed OS crash-reporting works and the new APIs associated with it.
+This example demonstrates how Mbed OS crash reporting works on an Mbed OS Enabled platform.
 
-**Table of contents:**
+You can find more information about the crash reporting APIs [in the documetnation](https://os.mbed.com/docs/mbed-os/latest/apis/error-handling.html#crash-reporting-and-auto-reboot).
 
-1. [Requirements](#requirements)
-1. [Setup](#setup)
-   - [Clone the example](#import-the-example)
-   - [Compile the example](#compile-the-example)
-   - [Run the example](#run-the-example)
+You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-## Requirements
+1. Install Mbed CLI.
+1. Clone this repository on your system.
+1. Change the current directory to where the project was cloned.
 
-This example program uses K64F as the test device.
+## Application functionality
 
+This example demonstrates the Mbed OS crash reporting feature and the APIs associated with it. In the first pass, the application generates a fault that causes the system to reboot. In the second pass, the application detects that the reboot was caused by an error and prints the fault context information in the console.
```

Comment:
Would it make sense to say "first part" or "first stage" instead of "first pass"?

Reply:
We can say first run instead.